dandiapi/api/tasks/__init__.py

```diff
@@ -157,10 +157,10 @@ def validate_version_metadata(version_id: int) -> None:
         version.validation_errors = [{'field': '', 'message': str(e)}]
         version.save()
         return
-    logger.info('Successfully validated version %s', version_id)
     version.status = Version.Status.VALID
     version.validation_errors = []
     version.save()
+    _log_version_action('Successfully validated dandiset metadata', version_id, version)
```

Comment:
note: here I also moved it down so we could time entire task execution, but if not desired -- can move it back up

Reply:
Is there a question of how long these tasks take? If not (i.e., there's no specific reason to wonder about this) then I would leave the log statement where it was.

Reply:
done